.circleci/config.yml

```diff
@@ -0,0 +1,62 @@
+version: 2.1
+
+references:
+  container_config: &container_config
+    docker:
+      - image: aeternity/infrastructure:Split-terraform-modules-in-separate-repos-PT-165295661
+    working_directory: /infrastructure
+
+  check_terraform_changes: &check_terraform_changes
+    run:
+      name: Fail on Terraform changes
+      command: make check-terraform-changes
+
+  setup_secrets: &setup_secrets
+    run:
+      name: Setup Environment
+      command: echo "source import-secrets.sh" >> $BASH_ENV
+
+commands:
+  integration_tests:
+    description: "Integration Tests"
+    steps:
+      - run:
+          name: Integration Tests
+          command: |
+            make integration-tests TF_VAR_envid="tf_test_${CIRCLE_JOB}_${CIRCLE_SHA1}"
+      - run:
+          name: Integration env cleanup
+          command: |
+            make integration-tests-cleanup TF_VAR_envid="tf_test_${CIRCLE_JOB}_${CIRCLE_SHA1}"
+          when: on_fail
+
+jobs:
+  check_terraform_changes:
+    <<: *container_config
+    steps:
+      - *setup_secrets
+      - run:
+          name: Fail on Terraform changes
+          command: |
+            make check-terraform-changes
+
+  # Checks the compatibility of latest package build and master bootstrap
+  integration_tests:
+    <<: *container_config
+    steps:
+      - *setup_secrets
+      - run:
+          name: Setup integration tests parameters
+          command: |
+            echo "export TF_VAR_bootstrap_version=v2.0.1" >> $BASH_ENV
+      - integration_tests
+
+workflows:
+  check_deploy:
+    jobs:
```

Comment:
you have to add `context:` to each job that needs secrets

Reply:
added in 9bd092f